pkg/services/pluginsintegration/pluginstore/plugins.go

```diff
@@ -10,7 +10,7 @@ import (
 type Plugin struct {
 	plugins.JSONData
 
-	fs                plugins.FS
+	FS                plugins.FS
```

Comment:
Nit. Feels like a leaky abstraction for a DTO, but understand maybe tricky to workaround/find a better solution? Maybe something to follow up on...